### ISSUE TITLE
Add pv plotting diagrams

### ISSUE THREAD

**Issue by keflavich** [radio-astro-tools/pvextractor#101]
We don't have any documentation on how to show PV diagram figures.

Super short version is:
```python
   pvfits = fits.open('extracted_pv.fits')
   ww = wcs.WCS(pvfits[0].header)
   ax = pl.subplot(111, projection=ww)
```

but there's a lot of subtlety in how to make it look good and change units etc.

**Comment by keflavich** [comment]
https://docs.astropy.org/en/stable/visualization/wcsaxes/controlling_axes.html is part of the answer, https://github.com/radio-astro-tools/tutorials/blob/6810376c0353f0bdf3be2b9b7231c388e886adba/PVDiagramPlotting.ipynb is another part

**Comment by keflavich** [comment]
also https://docs.astropy.org/en/stable/visualization/wcsaxes/slicing_datacubes.html

**Comment by keflavich** [comment]
closed by #102
